planner/core/logical_plan_builder.go

```diff
@@ -2611,6 +2614,13 @@ func (b *PlanBuilder) buildUpdateLists(tableList []*ast.TableName, list []*ast.A
 		col := assign.Col
 
 		dbName := col.DBName.L
+		// With sql below we need to get database by table alia name
```

Comment:
What the `alia` stand for?
```suggestion
		// With sql below, we need to get database by the table alias name.
```